src/app/containers/CpsAssetPageMain/index.jsx

```diff
@@ -26,7 +27,7 @@ const CpsAssetPageMain = ({ pageData }) => {
     subheadline: headings,
     text,
     image,
-    timestamp,
+    timestamp: allowDateStamp ? timestamp : undefined,
```

Comment:
`undefined` or `null`? Not sure which is more appropriate here

Reply:
Ahhh @dr3 suggeted `undefined` - https://github.com/bbc/simorgh/pull/4550#discussion_r342081553

Reply:
Is there any actual difference in terms of a react component?  Defining something as undefined just seems weird

Reply:
For this case I don't think there is a difference.  Semantically, `null` is more correct. There's a [check](https://github.com/bbc/simorgh/blob/latest/src/app/containers/Blocks/index.jsx#L14) on the component type, and so `null` and `undefined` will produce the same result. But `null` works here since the timestamp is *defined*. It's only nullified when `allowDateStamp` is false

Reply:
I figured `undefined` as effectively we dont want it defined, falling back on the default case, but yeah it doesn't matter really :P